### ISSUE TITLE
Conch cftp man page is broken

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3076]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#3076|
|Type|defect|
|Created|2008-03-15 22:35:28Z|

That should be fixed.

Attachments:

* [cftp_man.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/338/338f0d4fe9bbdb0ba989af906cf9caf9d18f326e/2cc0e694ce5cd7aec7b5eba1e27e111e7b2a9329.diff) (2560 bytes) - added by therve on 2008-03-15 22:35:57Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3076 3076
type__defect defect
reporter__therve therve
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__lore lore
keywords__ 
time__1205620528000000 1205620528000000
changetime__1205760032000000 1205760032000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)|@therve removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="dreid's avatar" src="https://avatars.githubusercontent.com/u/48695?s=50" width="50" height="50">](https://github.com/dreid)<a name="note_2"></a>|@dreid set owner to @therve|
|-|-|

Working within the existing framework this appears to be correct.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set status to `closed`|
|-|-|

(In [22860]) Fix management of the Interactive Command tag in man2lore: in particular,
cftp man page was broken after a recent on the list of options.

Author: therve
Reviewer: dreid
Fixes [#3076](https://github.com/twisted/twisted/issues/3076)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
